qlib/finco/task.py

```diff
@@ -246,24 +246,19 @@ def execute(self):
         if confirm is False:
             return []
 
-        model = init_instance_by_config(workflow["task"]["model"])
-        dataset = init_instance_by_config(workflow["task"]["dataset"])
-
-        with R.start(experiment_name="finCo"):
-            model.fit(dataset)
-            R.save_objects(trained_model=model)
-
-            # prediction
-            recorder = R.get_recorder()
-            sr = SignalRecord(model, dataset, recorder)
-            sr.generate()
-
-        self._context_manager.set_context("model", model)
-        self._context_manager.set_context("dataset", dataset)
-        self._context_manager.set_context("recorder", recorder)
+        command = f"qrun {workflow_path}"
+        self._output = subprocess.check_output(command, shell=True)
```

Comment:
Please set cwd to make sure the mlflow results are generated in the workspace.

Otherwise you can use CMDTask

Reply:
added cwd to check_output.